documentation/docs/80-adapter-api.md

```diff
@@ -23,15 +23,17 @@ export default function (options) {
 
 The types for `Adapter` and its parameters are available in [types/config.d.ts](https://github.com/sveltejs/kit/blob/master/packages/kit/types/config.d.ts).
 
-Within the `adapt` method, there are a number of things that an adapter should do:
+Within the `adapt` method, an adapter should
 
 - Clear out the build directory
-- Output code that:
-  - Calls `init`
-  - Converts from the platform's request to a [SvelteKit request](#hooks-handle), calls `render`, and converts from a [SvelteKit response](#hooks-handle) to the platform's
-  - Globally shims `fetch` to work on the target platform. SvelteKit provides a `@sveltejs/kit/install-fetch` helper for platforms that can use `node-fetch`
+- Import `{ appResolver }` from `@sveltejs/kit/adapter`
+- Pass in `appResolver()` to esbuild plugins for build options
+- Import `app` from `@sveltejs/kit/app`
+- Calls `app.init()`
```

Comment:
Totally missed those! Agreed it would read better with the "Output code that" part, it might be better to make this its own section too as it's getting long.